accounts/views.py

```diff
@@ -33,7 +36,31 @@ def activate(request, activation_key):
     profile = get_object_or_404(User, activation_key=activation_key)
     if profile.akey_expires < timezone.now():
         return render_to_response('accounts/activate.html', {'expired': True})
-    profile.is_active = True
+    profile.active = True
     profile.activation_key = ''
     profile.save()
     return render_to_response('accounts/activate.html', {'success': True})
+
+
+class LogInView(generic.FormView):
+    template_name = "accounts/login.html"
+    form_class = LogInForm
+
+    def form_valid(self, form):
+        email = form.cleaned_data['email']
+        password = form.cleaned_data['password']
+        user = authenticate(email=email, password=password)
+        if user.active:
+            login(self.request, user)
+        return render_to_response('accounts/home.html', {'success': True})
```

Comment:
Jeżeli formularz się poprawnie waliduje to powinieneś raczej robić redirecta. Zapobiega to ponownemu wysyłaniu formularzy, po odświeżeniu dalej Ci działają urle itd.